internal/cmd/identities.go

```diff
@@ -319,17 +315,12 @@ func checkPasswordRequirements(oldPassword string) survey.Validator {
 	}
 }
 
-func checkConfirmPassword(password *string) survey.Validator {
+func checkConfirmPassword() survey.Validator {
```

Comment:
I think this function should be removed entirely

Reply:
remove the type check?

Reply:
Yeah, anything the survey library lets them enter should be a valid string (it doesn't allow nil response)